torchx/workspace/docker_workspace.py

```diff
@@ -93,6 +93,12 @@ def workspace_opts(self) -> runopts:
             type_=str,
             help="(remote jobs) the image repository to use when pushing patched images, must have push access. Ex: example.com/your/container",
         )
+        opts.add(
+            "verbose",
+            type_=bool,
+            default=False,
+            help="(remote jobs) whether to print detailed output. Defaults to ``False``.",
```

Comment:
can we mention that this is specifically for verbose docker logs? I think we just merge these options normally